src/Homebrew.jl

```diff
@@ -16,7 +16,11 @@ const brew_prefix = abspath(joinpath(dirname(@__FILE__),"..","deps", "usr"))
 const brew_exe = joinpath(brew_prefix,"bin","brew")
 const tappath = joinpath(brew_prefix,"Library","Taps","staticfloat","homebrew-juliadeps")
 
-const BREW_URL = "https://github.com/Homebrew/homebrew.git"
+if VERSION < v"0.5.0-dev+522"
+    const BREW_URL = "https://github.com/Homebrew/homebrew.git"
+else
+    const BREW_URL = "https://github.com/Homebrew/brew.git"
+end
```

Comment:
Why are we doing a `VERSION` check here?  Is there some Julia feature we're using here? I think it'd be better to just make this change on all Julia versions after lots of testing.